src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -599,9 +599,13 @@ ucs_status_t uct_rc_mlx5_ep_flush(uct_ep_h tl_ep, unsigned flags,
                                                UCT_FLUSH_FLAG_REMOTE),
                     "flush flags CANCEL and REMOTE are mutually exclusive");
 
-    if ((flags & UCT_FLUSH_FLAG_REMOTE) &&
-        uct_rc_ep_is_flush_remote(&ep->super)) {
-        return uct_rc_mlx5_ep_flush_remote(tl_ep, comp);
+    if (flags & UCT_FLUSH_FLAG_REMOTE) {
+        UCT_CHECK_PARAM(uct_rc_iface_is_flush_remote(&iface->super),
+                        "iface %p is not configured to process flush(remote) "
+                        "op", iface);
+        if (uct_rc_ep_is_flush_remote(&ep->super)) {
+            return uct_rc_mlx5_ep_flush_remote(tl_ep, comp);
+        }
```

Comment:
need to align RC and DC code so they will look similar:
- DC checks UCT_DC_MLX5_EP_FLAG_FLUSH_RKEY and UCT_DC_MLX5_EP_FLAG_FLUSH_REMOTE
- RC checks via uct_rc_ep_is_flush_remote(), which is checking UCT_RC_EP_FLAG_FLUSH_REMOTE and  uct_ib_md_is_flush_rkey_valid(ep->flush_rkey)

Reply:
> need to align RC and DC code so they will look similar:

not sure I got your point... added similar routine into dc scope: uct_dc_ep_is_flush_remote